xc/domain.ml

```diff
@@ -351,7 +351,7 @@ let shutdown ~xc ~xs domid req =
 
 (** If domain is PV, signal it to shutdown. If the PV domain fails to respond then throw a Watch.Timeout exception.
 	All other exceptions imply the domain has disappeared. *)
-let shutdown_wait_for_ack (t: Xenops_task.task_handle) ?(timeout=60.) ~xc ~xs domid req =
+let shutdown_wait_for_ack (t: Xenops_task.task_handle) ?(timeout=(!Resources.domain_shutdown_ack_timeout)) ~xc ~xs domid req =
```

Comment:
I do wonder if this is the right thing to do here, design-wise, or if it is best to keep a hard-coded value? FYI: This function is currently only called from `Xenops_server_xen.request_shutdown`...

Reply:
I don't like this very much but it makes the code cleaner. Let's wait for @robhoes opinion

Reply:
We can just make the argument non-optional.